diff --git a/build/tasks/kernel.mk b/build/tasks/kernel.mk
index 8965b1c..e5e9c41 100644
--- a/build/tasks/kernel.mk
+++ b/build/tasks/kernel.mk
@@ -321,7 +321,8 @@ INSTALLED_KERNEL_MODULES: depmod-host
 			echo "Installing Kernel Modules"; \
 			$(MAKE) $(MAKE_FLAGS) -C $(KERNEL_SRC) O=$(KERNEL_OUT) ARCH=$(KERNEL_ARCH) $(KERNEL_CROSS_COMPILE) $(KERNEL_CLANG_TRIPLE) $(KERNEL_CC) INSTALL_MOD_PATH=../../$(KERNEL_MODULES_INSTALL) modules_install && \
 			mofile=$$(find $(KERNEL_MODULES_OUT) -type f -name modules.order) && \
-			mpath=$$(dirname $$mofile) && \
+			#mpath=$$(dirname $$mofile) && \
+			mpath="" && \
 			for f in $$(find $$mpath/kernel -type f -name '*.ko'); do \
 				$(KERNEL_TOOLCHAIN_PATH)strip --strip-unneeded $$f; \
 				mv $$f $(KERNEL_MODULES_OUT); \
